### PR TITLE
Chart range

### PR DESCRIPTION
**Pull request by mathjazz**
@jotes r?

This is a request for the UX testing only.

Code is not ready yet, I still need to remove the old date picker.

### CODE REVIEW

File: pontoon/base/models.py [hunk 1]
```diff
@@ -1371,6 +1371,32 @@ def find_and_replace(self, find, replace, user):
...
+            .extra({'minute': "date_trunc('minute', date)"})
+            .order_by('minute')
+            .values('minute')
+            .annotate(count=Count('id'))
+        )
+
+        data = []
+        for period in translations:
+            data.append([
+                utils.convert_to_unix_time(period['minute']),
```
Comment (jotes) [inline pontoon/base/models.py]: I think we could replace this loop with time conversion inside the sql query and  .values_list() call at the end.

Comment (mathjazz) [inline pontoon/base/models.py]: We need a list of lists, but the `.values_list()` only gives us a list of tuples I think.

**Comment by jotes** [comment]
My initial impressions: whoa, this looks awesome. I can't wait until buttons will be fully working. This looks awesome. I'll try to put some more data and stress this widget a little.

**Comment by mathjazz** [comment]
@jotes r?

**Comment by jotes** [comment]
r+ :-)

**Pull request merged**